cmd/template-resolver/utils/resolver_utils.go

```diff
@@ -191,20 +193,25 @@ func ProcessTemplate(yamlBytes []byte, hubKubeConfigPath, clusterName, hubNS str
 		return nil, fmt.Errorf("failed to instantiate the template resolver: %w", err)
 	}
 
+	tempCtx := templates.TemplateContext{
+		ObjectNamespace: objNamespace,
+		ObjectName:      objName,
+	}
+
 	switch policy.GetKind() {
 	case "Policy":
-		err = processPolicyTemplate(&policy, resolver)
+		err = processPolicyTemplate(&policy, resolver, tempCtx)
 	case "ConfigurationPolicy":
-		err = processConfigPolicyTemplate(&policy, resolver)
+		err = processConfigPolicyTemplate(&policy, resolver, tempCtx)
 	case "OperatorPolicy":
-		_, err = processOperatorPolicyTemplates(policy.Object, resolver)
+		_, err = processOperatorPolicyTemplates(policy.Object, resolver, tempCtx)
 	default:
 		if _, ok := policy.Object["object-templates-raw"]; !ok {
 			return nil, fmt.Errorf("invalid YAML. Supported types: Policy, " +
 				"ConfigurationPolicy, object-templates-raw")
```

Comment:
I know this isn't in your change, but I can't help but notice that OperatorPolicy is supported but not included in this error message

```suggestion
			return nil, fmt.Errorf("invalid YAML. Supported types: Policy, " +
				"ConfigurationPolicy, OperatorPolicy, object-templates-raw")
```